src/pds/llm/Wiki2Vec.py

```diff
@@ -28,66 +49,51 @@ def get_embeddings(url, tokenizer=word_tokenize_pds4_xml_files):
     return vectors
 
 
+def get_search_embeddings(term):
+    """
```

Comment:
Docstrings should start immediately touching the `"""`, should start with a capital letter, and start with a single sentence ending with a period. The rest of the docstring can go into more detail and is more freeform.